test/unit.hs

```diff
@@ -171,7 +171,7 @@ conformanceTestsFor weighting fp = do
                      [] -> return ()
                      es -> assertFailure (unlines es))
          $ map (conformanceTestWith coll)
-              (zip3 (map fst xs) (map snd xs) (tail (map snd xs)))
+              (zip3 (map fst xs) (map snd xs) (drop 1 (map snd xs)))
```

Comment:
GHC 9.8 now warns about using Data.List.tail, which becomes an error with the `-Wall` flag